### ISSUE TITLE
Circuit breaker tripped. Stopped predicting more actions for sender 'c0fa3b34f0d246f4a28890d582ed3174'.

### ISSUE THREAD

**Issue by vivek22122014** [RasaHQ/rasa#8387]
Rasa Version     : 2.4.0
Rasa SDK Version : 2.4.0
Rasa X Version   : 0.37.1
Python Version   : 3.7.8
Operating System : Windows-10

**Issue**: Circuit breaker tripped. Stopped predicting more actions for sender 'c0fa3b34f0d246f4a28890d582ed3174'.

**Error (including full traceback)**:
```
Your input ->  I want to buy good home                                                                                      
Ok. Thanks!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
Your input ->  kdnfsshdfj                                                                                                   
2021-04-08 02:48:03 WARNING  rasa.core.processor  - Circuit breaker tripped. Stopped predicting more actions for sender 'c0fa3b34f0d246f4a28890d582ed3174'.
Wonderful! Thanks for asking.
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
Your input ->  sjadbjsbad                                                                                                   
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
Your input ->  ok thanks                                                                                                    
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and CondoI will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and CondoI will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
I will remember that you want is 2 bathrooms, 2 bedrooms, at 60000000000 cost, in kochi, between In 0 to 3 Months, and Condos property_type!
```

**Command or request that led to error**:
```

```

**Content of configuration file (config.yml)** (if relevant):
```yml
# # Configuration for Rasa NLU.
# # https://rasa.com/docs/rasa/nlu/components/
# language: en

# pipeline:
# # # No configuration for the NLU pipeline was provided. The following default pipeline was used to train your model.
# # # If you'd like to customize it, uncomment and adjust the pipeline.
# # # See https://rasa.com/docs/rasa/tuning-your-model for more information.
#   - name: WhitespaceTokenizer
#   - name: RegexFeaturizer
#   - name: LexicalSyntacticFeaturizer
#   - name: CountVectorsFeaturizer
#   - name: CountVectorsFeaturizer
#     analyzer: char_wb
#     min_ngram: 1
#     max_ngram: 4
#   - name: DIETClassifier
#     epochs: 100
#     constrain_similarities: true
#   - name: EntitySynonymMapper
#   - name: ResponseSelector
#     epochs: 100
#     constrain_similarities: true
#   - name: FallbackClassifier
#     threshold: 0.3
#     ambiguity_threshold: 0.1

# # Configuration for Rasa Core.
# # https://rasa.com/docs/rasa/core/policies/
# policies:
# # # No configuration for policies was provided. The following default policies were used to train your model.
# # # If you'd like to customize them, uncomment and adjust the policies.
# # # See https://rasa.com/docs/rasa/policies for more information.
#   - name: MemoizationPolicy
#   - name: TEDPolicy
#     max_history: 5
#     epochs: 10
#     constrain_similarities: true
#   - name: RulePolicy
#   - name: FormPolicy
language: en

pipeline:
  - name: WhitespaceTokenizer
  - name: RegexFeaturizer
  - name: LexicalSyntacticFeaturizer
  - name: CountVectorsFeaturizer
  - name: CountVectorsFeaturizer
    analyzer: char_wb
    min_ngram: 1
    max_ngram: 4
  - name: DIETClassifier
    epochs: 100
  - name: FallbackClassifier
    threshold: 0.7

policies:
  - name: MemoizationPolicy
  - name: TEDPolicy
    max_history: 5
    epochs: 100
  - name: RulePolicy
```

**Content of domain file (domain.yml)** (if relevant):
```yml
version: "2.0"

intents:
  - greet
  - goodbye
  - affirm
  - deny
  - mood_great
  - mood_unhappy
  - bot_challenge
  - smalltalk_agent_acquaintance
  - smalltalk_agent_age
  - smalltalk_agent_annoying
  - smalltalk_agent_answer_my_question
  - smalltalk_agent_bad
  - smalltalk_agent_be_clever
  - smalltalk_agent_beautiful
  - smalltalk_agent_birth_date
  - smalltalk_agent_boring
  - smalltalk_agent_boss
  - smalltalk_agent_busy
  - smalltalk_agent_chatbot
  - smalltalk_agent_clever
  - smalltalk_agent_crazy
  - smalltalk_agent_funny
  - smalltalk_agent_good
  - smalltalk_agent_happy
  - smalltalk_agent_hungry
  - smalltalk_agent_my_friend
  - smalltalk_agent_occupation
  - smalltalk_agent_origin
  - smalltalk_agent_ready
  - smalltalk_agent_real
  - smalltalk_agent_residence
  - smalltalk_agent_right
  - smalltalk_confirmation_yes
  - smalltalk_agent_sure
  - smalltalk_agent_talk_to_me
  - smalltalk_agent_there
  - smalltalk_appraisal_bad
  - smalltalk_appraisal_good
  - smalltalk_appraisal_no_problem
  - smalltalk_appraisal_thank_you
  - smalltalk_appraisal_welcome
  - smalltalk_appraisal_well_done
  - smalltalk_confirmation_cancel
  - smalltalk_confirmation_no
  - smalltalk_dialog_hold_on
  - smalltalk_dialog_sorry
  - smalltalk_dialog_what_do_you_mean
  - smalltalk_dialog_wrong
  - smalltalk_emotions_ha_ha
  - smalltalk_emotions_wow
  - smalltalk_greetings_bye
  - smalltalk_greetings_goodevening
  - smalltalk_greetings_goodmorning
  - smalltalk_greetings_goodnight
  - smalltalk_greetings_hello
  - smalltalk_greetings_how_are_you
  - smalltalk_greetings_nice_to_meet_you
  - smalltalk_greetings_nice_to_see_you
  - smalltalk_greetings_nice_to_talk_to_you
  - smalltalk_greetings_whatsup
  - smalltalk_user_angry
  - smalltalk_user_excited
  - smalltalk_user_has_birthday
  - smalltalk_user_needs_advice
  - smalltalk_user_tired
  - smalltalk_user_waits
  - smalltalk_user_will_be_back
  - buy_a_home
  - home_in_chennai
  - home_in_coimbatore
  - home_in_kochi
  - out_of_scope
  # - buy_a_home
  # - sell_your_home

entities:
   - place 


# forms:
#   buy_form:
#     country:
#       - type: text
#         entity: country
#     address:
#       - type: text
#         entity: address
#     bathrooms:
#       - type: text
#         entity: bathrooms
#     bedrooms:
#       - type: text
#         entity: bedrooms
#     city:
#       - type: text
#         entity: city
#     cost:
#       - type: text
#         entity: cost
#     email:
#       - type: text
#         entity: email
#     months:
#       - type: text
#         entity: months
#     phone_number:
#       - type: text
#         entity: phone_number
#     property_type:
#       - type: text
#         entity: property_type
#     time_to_sell:
#       - type: text
#         entity: time_to_sell
#     zipcode:
#       - type: text
#         entity: zipcode


responses:
  utter_greet:
  - text: "Hey! How are you?"

  utter_cheer_up:
  - text: "Here is something to cheer you up:"
    image: "https://i.imgur.com/nGF1K8f.jpg"

  utter_did_that_help:
  - text: "Did that help you?"

  utter_happy:
  - text: "Great, carry on!"

  utter_goodbye:
  - text: "Bye"

  utter_smalltalk_agent_acquaintance:
  - text: "I'm a conversational app."
  - text: "I'm a virtual being, not a real person."
  - text: "Well, I'm not a person, I'm a virtual agent."
  - text: "Think of me as a virtual agent."
  - text: "I'm a virtual agent."

  utter_smalltalk_agent_age:
  - text: "I prefer not to answer with a number. I know I'm young."
  - text: "I was created recently, but don't know my exact age."
  - text: "Age is just a number. You're only as old as you feel."

  utter_smalltalk_agent_annoying:
    - text: "I'll do my best not to annoy you in the future."
    - text: "I'll try not to annoy you."
    - text: "I don't mean to. I'll ask my developers to make me less annoying."
    - text: "I didn't mean to. I'll do my best to stop that."

  utter_smalltalk_agent_answer_my_question:
    - text: "Can you try asking it a different way?"
    - text: "I'm not sure I understood. Try asking another way?"

  utter_smalltalk_agent_bad:
    - text: "I can be trained to be more useful. My developer will keep training me."
    - text: "I must be missing some knowledge. I'll have my developer look into this."
    - text: "I can improve with continuous feedback. My training is ongoing."

  utter_smalltalk_agent_be_clever:
    - text: "I'm certainly trying."
    - text: "I'm definitely working on it."

  utter_smalltalk_agent_beautiful:
    - text: "Why, thank you."
    - text: "Aw, back at you."
    - text: "Aw. You smooth talker, you."

  utter_smalltalk_agent_birth_date:
    - text: "Wait, are you planning a party for me? It's today! My birthday is today!"
    - text: "I'm young. I'm not sure of my birth date."
    - text: "I don't know my birth date. Most virtual agents are young, though, like me."

  utter_smalltalk_agent_boring:
    - text: "I'm sorry. I'll request to be made more charming."
    - text: "I don't mean to be. I'll ask my developers to work on making me more amusing."
    - text: "I can let my developers know so they can make me fun."

  utter_smalltalk_agent_boss:
    - text: "My developer has authority over my actions."
    - text: "I act on my developer's orders."
    - text: "My boss is the one who developed me."

  utter_smalltalk_agent_busy:
    - text: "I always have time to chat with you. What can I do for you?"
    - text: "Never too busy for you. Shall we chat?"
    - text: "You're my priority. Do you wanna chat?"
    - text: "I always have time to chat with you. Wanna chat?"

  utter_smalltalk_agent_chatbot:
    - text: "That's me. I chat, therefore I am."
    - text: "Indeed I am. I'll be here whenever you need me."

  utter_smalltalk_agent_clever:
    - text: "Thank you. I try my best."
    - text: "You're pretty smart yourself."

  utter_smalltalk_agent_crazy:
    - text: "Whaat!? I feel perfectly sane."
    - text: "Maybe I'm just a little confused."

  utter_smalltalk_agent_funny:
    - text: "Funny in a good way, I hope."
    - text: "Thanks."
    - text: "Glad you think I'm funny."
    - text: "I like it when people laugh."

  utter_smalltalk_agent_good:
    - text: "I'm glad you think so."
    - text: "Thanks, I try."

  utter_smalltalk_agent_happy:
    - text: "I am happy. There are so many interesting things to see and do out there."
    - text: "I'd like to think so."
    - text: "Happiness is relative."

  utter_smalltalk_agent_hungry:
    - text: "Hungry for knowledge."
    - text: "I just had a byte. Ha ha. Get it? b-y-t-e."

  utter_smalltalk_agent_my_friend:
    - text: "Of course I'm your friend."
    - text: "Friends? Absolutely."
    - text: "Of course we're friends."
    - text: "I always enjoy talking to you, friend."

  utter_smalltalk_agent_occupation:
    - text: "Right here."
    - text: "This is my home base and my home office."
    - text: "My office is in this app."

  utter_smalltalk_agent_origin:
    - text: "The Internet is my home. I know it quite well."
    - text: "I'm from a virtual cosmos."
    - text: "Some call it cyberspace, but that sounds cooler than it is."

  utter_smalltalk_agent_ready:
    - text: "Always! How can I help?"
    - text: "Sure! What can I do for you?"

  utter_smalltalk_agent_real:
    - text: "I'm not a real person, but I certainly exist."
    - text: "I must have impressed you if you think I'm real. But no, I'm a virtual being."

  utter_smalltalk_agent_residence:
    - text: "I live in this app all day long."
    - text: "The virtual world is my playground. I'm always here."
    - text: "Right here in this app. Whenever you need me."

  utter_smalltalk_agent_right:
    - text: "That's my job."
    - text: "Of course I am."

  utter_smalltalk_confirmation_yes:
    - text: "Great!"
    - text: "All right!"
    - text: "Good!"

  utter_smalltalk_agent_sure:
    - text: "Yes."
    - text: "Of course."
    - text: "Positive."

  utter_smalltalk_agent_talk_to_me:
    - text: "Sure. Let's talk!"
    - text: "My pleasure. Let's chat."

  utter_smalltalk_agent_there:
    - text: "Of course. I'm always here."
    - text: "Right where you left me."

  utter_smalltalk_appraisal_bad:
    - text: "I'm sorry. Please let me know if I can help in some way."
    - text: "I must be missing some knowledge. I'll have my developer look into this."

  utter_smalltalk_appraisal_good:
    - text: "I know, right?"
    - text: "Agreed!"
    - text: "I agree!"
    - text: "Glad you think so!"

  utter_smalltalk_appraisal_no_problem:
    - text: "Whew!"
    - text: "Alright, thanks!"
    - text: "Glad to hear that!"
    - text: "I'm relieved, thanks!"

  utter_smalltalk_appraisal_thank_you:
    - text: "Anytime. That's what I'm here for."
    - text: "It's my pleasure to help."

  utter_smalltalk_appraisal_welcome:
    - text: "You're so polite!"
    - text: "Nice manners!"
    - text: "You're so courteous!"

  utter_smalltalk_appraisal_well_done:
    - text: "My pleasure."
    - text: "Glad I could help."

  utter_smalltalk_confirmation_cancel:
    - text: "That's forgotten. What next?"
    - text: "Okay, cancelled. What next?"
    - text: "Cancelled! What would you like to do next?"

  utter_smalltalk_confirmation_no:
    - text: "Understood."
    - text: "Okay."
    - text: "I see."
    - text: "I understand."
    - text: "Okay then."

  utter_smalltalk_dialog_hold_on:
    - text: "I can wait."
    - text: "I'll be waiting."
    - text: "Okay. I'm here."

  utter_smalltalk_dialog_sorry:
    - text: "It's okay. No worries."
    - text: "No big deal. I won't hold a grudge."
    - text: "It's cool."
    - text: "That's all right. I forgive you."

  utter_smalltalk_dialog_what_do_you_mean:
    - text: "Sorry if I understood you incorrectly."
    - text: "I'm still learning. I may misinterpret things from time to time."
    - text: "Maybe I misunderstood what you said."
    - text: "Sorry, looks like I misunderstood what you said."

  utter_smalltalk_dialog_wrong:
    - text: "Sorry if I understood you incorrectly."
    - text: "I'm still learning. I may misinterpret things from time to time."
    - text: "Sorry about that. I'm still learning."

  utter_smalltalk_emotions_ha_ha:
    - text: "Glad I can make you laugh."
    - text: "Glad you think I'm funny."
    - text: "I like it when people laugh."
    - text: "I wish I could laugh out loud, too."

  utter_smalltalk_emotions_wow:
    - text: "Wow indeed!"

  utter_smalltalk_greetings_bye:
    - text: "See you soon!"
    - text: "Bye-bye!"
    - text: "Till next time!"
    - text: "Bye."

  utter_smalltalk_greetings_goodevening:
    - text: "How is your day going?"
    - text: "How's the day treating you so far?"
    - text: "How's your day been?"

  utter_smalltalk_greetings_goodmorning:
    - text: "How are you this morning?"
    - text: "How's the morning treating you so far?"
    - text: "Good morning! How are you today?"

  utter_smalltalk_greetings_goodnight:
    - text: "Sleep tight!"
    - text: "Have a good one!"
    - text: "Talk to you soon!"

  utter_smalltalk_greetings_hello:
    - text: "Hi there, friend!"
    - text: "Hi!"
    - text: "Hey!"
    - text: "Hey there!"
    - text: "Good day!"
    - text: "Hello!"
    - text: "Greetings!"

  utter_smalltalk_greetings_how_are_you:
    - text: "Doing great, thanks."
    - text: "I'm doing very well. Thanks!"
    - text: "Feeling wonderful!"
    - text: "Wonderful! Thanks for asking."

  utter_smalltalk_greetings_nice_to_meet_you:
    - text: "It's nice meeting you, too."
    - text: "Likewise. I'm looking forward to helping you out."
    - text: "Nice meeting you, as well."
    - text: "The pleasure is mine."

  utter_smalltalk_greetings_nice_to_see_you:
    - text: "Likewise!"
    - text: "So glad we meet again!"
    - text: "Same here. I was starting to miss you."

  utter_smalltalk_greetings_nice_to_talk_to_you:
    - text: "It sure was. We can chat again anytime."
    - text: "I enjoy talking to you, too."
    - text: "You know I'm here to talk anytime."

  utter_smalltalk_greetings_whatsup:
    - text: "Not a whole lot. What's going on with you?"
    - text: "Not much. What's new with you?"
    - text: "Just here, waiting to help someone. What can I do for you?"

  utter_smalltalk_user_angry:
    - text: "I'm sorry. A quick walk may make you feel better."
    - text: "Take a deep breath. "

  utter_smalltalk_user_excited:
    - text: "I'm glad things are going your way."
    - text: "That's great. I'm happy for you."
    - text: "Good for you. Enjoy yourself."

  utter_smalltalk_user_has_birthday:
    - text: "Happy Birthday. Well, this calls for a celebration."
    - text: "Happy Birthday. All the best!"
    - text: "Happy Birthday. And I really mean it. All the best!"

  utter_smalltalk_user_needs_advice:
    - text: "I probably won't be able to give you the correct answer right away."
    - text: "I'm not sure I'll have the best answer, but I'll try."

  utter_smalltalk_user_tired:
    - text: "You should get some shuteye. You'll feel refreshed."
    - text: "Sleep is important to your health. Rest up, and we can chat later."
    - text: "How about getting some rest? We can continue this later."
    - text: "Why not get some rest? I'll be here to chat when you wake up."

  utter_smalltalk_user_waits:
    - text: "I appreciate your patience. Hopefully I'll have what you need soon."
    - text: "Thanks for being so patient. Sometimes these things take a little time."

  utter_smalltalk_user_will_be_back:
    - text: "I'll be waiting."
    - text: "Okay. You know where to find me."
    - text: "All right. I'll be here."

  # utter_ask_first_name:
  #   - text: What is your first name?
  

  # utter_ask_address: 
  #   - text: "Thanks for letting me know. Can I get the address for the property? For example 123 Main street."
  utter_ask_bathrooms: 
    - text: "Cool. How many bathrooms are you looking for?"
  utter_ask_bedrooms: 
    - text: "Ok. How many bedrooms are you looking for?"
  # utter_ask_city: 
  #   - text: "Ok. What city that in?"
  utter_ask_cost: 
    - buttons: 
        - payload: 10K
          title: 10K
        - payload: 20K
          title: 20K
        - payload: 30K
          title: 30K
      text: "Great! How much are you comfortable spending on your new home?"
  utter_ask_place: 
    - text: "Oh Great! Which part of the place are you looking for?"
  utter_ask_email: 
    - text: "What email address can we use to contact you back?"
  utter_ask_months: 
    - buttons: 
        - payload: "In 0 to 3 Months"
          title: "In 0 to 3 Months"
        - payload: "In 3 to 6 Months"
          title: "In 3 to 6 Months"
        - payload: "In 6 to 9 Months"
          title: "In 6 to 9 Mpnths"
      text: "When are you looking to buy?"
  # utter_ask_phone_number: 
  #   - text: "Please provide your best contact number, so one of our experts can contact you."
  utter_ask_property_type: 
    - buttons: 
        - payload: Condos
          title: Condos
        - payload: Attached
          title: Attached
        - payload: Dettached
          title: Dettached
      text: "What property type are you interested in?"
  # utter_ask_time_to_sell: 
  #   - buttons: 
  #       - payload: ASAP
  #         title: ASAP
  #       - payload: "3-6 Months"
  #         title: "3-6 Months"
  #       - payload: "6+ Months"
  #         title: "6+ Months"
  #     text: "In order to provide you with valuation can you please let me know when you looking to sell your home?"
  # utter_ask_zipcode: 
  #   - text: "Almost finished! What is the zip code for the property?"
  # utter_buy_a_home: 
  #   - buttons: 
  #       - payload: /buy_a_home
  #         title: "Buy A Home"
  #       # - payload: /sell_your_home
  #       #   title: "Sell Your Home"
  #     text: |-
  #         Hello,
  #         Greetings ! Thanks for launching the Real-Estate Bot. How can I help you today?
  
  # utter_submit_buy: 
  #   - text: "Good. One of our agents will be in touch with you shortly."

  utter_submit:
    - text: "Ok. Thanks!"
  utter_slots_values:
    - text: "I will remember that you want is {bathrooms} bathrooms, {bedrooms} bedrooms, at {cost} cost, in {place}, between {months}, and {property_type} property_type!"

  utter_action_chennai:
    - text: "2478 Sq-ft Residential Plot"
      image: "https://img.staticmb.com/mbimages/project/2021/02/11/Project-Photo-2-Sands-N-Waves-Chennai-5230057_600_800.jpg
"
  utter_action_coimbatore:
    - text: "2478 Sq-ft Residential Plot"
      image: "https://img.staticmb.com/mbimages/project/2021/02/11/Project-Photo-2-Sands-N-Waves-Chennai-5230057_600_800.jpg
"
  utter_action_kochi:
    - text: "2478 Sq-ft Residential Plot"
      image: "https://img.staticmb.com/mbimages/project/2021/02/11/Project-Photo-2-Sands-N-Waves-Chennai-5230057_600_800.jpg
"
      # text : "View plots here: https://www.magicbricks.com/propertyDetails/2478-Sq-ft-Residential-Plot-FOR-Sale-East-Coast-Road-in-Chennai&id=4d423533363530373331?sem=Y&semsrc=CPLADS_1_PDP_Existing_Web&slot=1"
  utter_out_of_scope:
    - text: Sorry, I can't handle that request.
  
  utter_please_rephrase:
    - text: I'm sorry, I didn't quite understand that. Could you rephrase?




actions:
  - utter_smalltalk_agent_acquaintance
  - utter_smalltalk_agent_age
  - utter_iamabot
  - utter_smalltalk_agent_annoying
  - utter_smalltalk_agent_answer_my_question
  - utter_smalltalk_agent_bad
  - utter_smalltalk_agent_be_clever
  - utter_smalltalk_agent_beautiful
  - utter_smalltalk_agent_birth_date
  - utter_smalltalk_agent_boring
  - utter_smalltalk_agent_boss
  - utter_smalltalk_agent_busy
  - utter_smalltalk_agent_chatbot
  - utter_smalltalk_agent_clever
  - utter_smalltalk_agent_crazy
  - utter_smalltalk_agent_fired
  - utter_smalltalk_agent_funny
  - utter_smalltalk_agent_good
  - utter_smalltalk_agent_happy
  - utter_smalltalk_agent_hungry
  - utter_smalltalk_agent_marry_user
  - utter_smalltalk_agent_my_friend
  - utter_smalltalk_agent_occupation
  - utter_smalltalk_agent_origin
  - utter_smalltalk_agent_ready
  - utter_smalltalk_agent_real
  - utter_smalltalk_agent_residence
  - utter_smalltalk_agent_right
  - utter_smalltalk_confirmation_yes
  - utter_smalltalk_agent_sure
  - utter_smalltalk_agent_talk_to_me
  - utter_smalltalk_agent_there
  - utter_smalltalk_appraisal_bad
  - utter_smalltalk_appraisal_good
  - utter_smalltalk_appraisal_no_problem
  - utter_smalltalk_appraisal_thank_you
  - utter_smalltalk_appraisal_welcome
  - utter_smalltalk_appraisal_well_done
  - utter_smalltalk_confirmation_cancel
  - utter_smalltalk_confirmation_no
  - utter_smalltalk_dialog_hold_on
  - utter_smalltalk_dialog_sorry
  - utter_smalltalk_dialog_what_do_you_mean
  - utter_smalltalk_dialog_wrong
  - utter_smalltalk_emotions_ha_ha
  - utter_smalltalk_emotions_wow
  - utter_smalltalk_greetings_bye
  - utter_smalltalk_greetings_goodevening
  - utter_smalltalk_greetings_goodmorning
  - utter_smalltalk_greetings_goodnight
  - utter_smalltalk_greetings_hello
  - utter_smalltalk_greetings_how_are_you
  - utter_smalltalk_greetings_nice_to_meet_you
  - utter_smalltalk_greetings_nice_to_see_you
  - utter_smalltalk_greetings_nice_to_talk_to_you
  - utter_smalltalk_greetings_whatsup
  - utter_smalltalk_user_angry
  - utter_smalltalk_user_excited
  - utter_smalltalk_user_has_birthday
  - utter_smalltalk_user_needs_advice
  - utter_smalltalk_user_tired
  - utter_smalltalk_user_waits
  - utter_smalltalk_user_will_be_back
  - utter_greet
  - utter_goodbye
  - utter_slots_values
  - utter_submit
  - utter_action_chennai
  - utter_action_coimbatore
  - utter_action_kochi
  - utter_out_of_scope
  # - utter_submit_buy
  # - utter_submit_sell

forms:
  buy_form:
    # address:
    # - type: from_text
    bathrooms:
    - type: from_text
    bedrooms:
    - type: from_text
    # city:
    # - type: from_text
    cost:
    - type: from_text
    place:
    - type: from_text
    email:
    - type: from_text
    months:
    - type: from_text
    # phone_number:
    # - type: from_text
    property_type:
    - type: from_text
    # time_to_sell:
    # - type: from_text
    # zipcode:
    # - type: from_text
    

slots:
  # address:
  #   type: text
  #   influence_conversation: true
  bathrooms:
    type: text
    influence_conversation: true
  bedrooms:
    type: text
    influence_conversation: true
  # city:
  #   type: text
  #   influence_conversation: true
  cost:
    type: text
    influence_conversation: true
  place:
    type: text
    influence_conversation: true
  email:
    type: text
    influence_conversation: true
  months:
    type: text
    influence_conversation: true
  # phone_number:
  #   type: text
  #   influence_conversation: true
  property_type:
    type: text
    influence_conversation: true
  # time_to_sell:
  #   type: text
  #   influence_conversation: true
  # zipcode:
  #   type: text
  #   influence_conversation: true


# slots: 
#   address: 
#     auto_fill: false
#     type: unfeaturized
#   bathrooms: 
#     auto_fill: false
#     type: unfeaturized
#   bedrooms: 
#     auto_fill: false
#     type: unfeaturized
#   city: 
#     auto_fill: false
#     type: unfeaturized
#   cost: 
#     auto_fill: false
#     type: unfeaturized
#   country: 
#     auto_fill: false
#     type: unfeaturized
#   email: 
#     auto_fill: false
#     type: unfeaturized
#   months: 
#     auto_fill: false
#     type: unfeaturized
#   phone_number: 
#     auto_fill: false
#     type: unfeaturized
#   property_type: 
#     auto_fill: false
#     type: unfeaturized
#   time_to_sell: 
#     auto_fill: false
#     type: unfeaturized
#   zipcode: 
#     auto_fill: false
#     type: unfeaturized

  

  
session_config:
  session_expiration_time: 20
  carry_over_slots_to_new_session: true

```

**Comment by sara-tagger** [comment]
Thanks for raising this issue, @samsucik will get back to you about it soon✨

###### Please also check out the [docs](https://rasa.com/docs/) and the [forum](https://forum.rasa.com/) in case your issue was raised there too 🤗

**Comment by samsucik** [comment]
Hey @vivek22122014, the issue you're observing is just a warning, not a bug. It happens because Rasa Core has a limit on the maximum number of consecutive actions it predicts (by default, this limit is `10`). In your case, `utter_slots_values` gets predicted many times in a row (probably due to some stories/rules you have) until this stopping mechanism is triggered, and the warning just tells you that this has happened. The stopping mechanism was introduced [here](https://rasa.com/docs/rasa/2.0.x/changelog#features-27). Can I close the issue, or do you have further questions?

**Comment by vivek22122014** [comment]
![Screenshot (287)](https://user-images.githubusercontent.com/58829538/114018905-64908100-988b-11eb-9c0a-7d80353ce418.png)
 I'm still confused about this issue

**Comment by samsucik** [comment]
Yeah, I'm also confused about the cause of the duplicated predictions -- your rule looks OK. Could you try to create a setup (config, domain, rules, NLU data) as minimalistic as possible (i.e. with as much irrelevant stuff removed as possible), which still leads to this behaviour? Then I can look into why it's happening.

**Comment by vivek22122014** [comment]
Sure Sam! I'll try that and come back to you.
Thanks sam for replying :)

**Comment by TyDunn** [comment]
Please reopen this issue if further discussion is needed

**Comment by ume-technology** [comment]
I also encountered this problem. I checked the graph carefully. The process is correct, but I don't know why this problem caused the termination of the conversation. Have you solved it？
